app/dist/scripts/app.js

```diff
@@ -36,13 +36,8 @@ document.querySelectorAll("a[href^='#']").forEach(anchor => {
     if (document.getElementById(element)) {
       elementOffset = document.getElementById(element).offsetTop - 74;
       window.scroll({ top: elementOffset, behavior: "smooth" });
+      history.replaceState({}, "", `#${element}`); // Add hash to URL bar
     }
-
```

Comment:
Does removing this block affect other pages? For example, if I click a glossary link from the Overview page, will the smooth scroll and anchoring happen?

Reply:
I'm not sure how to checkout your branch to test myself.

Reply:
@Wick ^

Reply:
Tested on the overview page + api page and it smooth scrolls as it should and updates the URL. My reasoning behind the change was that if it manages to successfully find an element to scroll to it can safely be added to browser history.

I noticed a related issue though: that when smooth-scrolling occurs it wont get properly placed in browser history, on current site as well (navigate with the sidebar then click back in the browser).
My latest commit will fix this by changing from history.replaceState to history.pushState for updating browser history.

Let me know if you need anything more :)

Reply:
Sweet, sounds good to me. Thanks!